### ISSUE TITLE
Mongo-Cluster running without Password

### ISSUE THREAD

**Issue by appinteractive** [rancher/compose-templates#109]
How can I configure a password for my Mongo Cluster? At the default configuration its open to the wild! This sould really not be the default!

**Comment by sergiopellegrini** [comment]
@appinteractive Firstly setup your admin users on your mongo instance. Then upgrade your mongo cluster add --auth to your startup command.

**Comment by appinteractive** [comment]
Yeah I did it already and it worked. But to be honest that’s should be four fields to add a read write user and an admin user on first startup if not happened oder add a BIG warning to that to be aware of. 

I know it’s not your fault that mongo Standard config is unsafe bur we should not letting run other people into it. 

Hope you get my point and thanks for your package man! 🙂

**Comment by sergiopellegrini** [comment]
Fully agree with you! Not my package but was setting this up also and found it strange that there was no option to secure the DB.

**Comment by micw** [comment]
Looks like there's already a pull request for this: https://github.com/rancher/catalog-dockerfiles/pull/86 - but for some reason it's ignored...

**Comment by markqiu** [comment]
the cluseter has the following error after I add --auth:
Error in heartbeat request to 10.42.230.91:27017; Unauthorized: not authorized on admin to execute command { replSetHeartbeat: "rs0", configVersion: 5, from: "guchong-mongo-cluster-1:27017", fromId: 0, term: 2 }
Any idea?
